FixMyBerlin/fixmy.platform
# Add admin command to send notifications

- Add admin command to send notifications
  - Text and HTML variants
  - Combine multiple notices into one email
- Add tests for that command
- Add env var `EMAIL_REPLY_TO`
- Add helper methods to provide unsubscribe_url and user notification preference on StatusNotice model

Various
- Add debug output to `importreportplannings` command

Closes #453

## How Has This Been Tested?

- Create a report with an associated user
- Change that report's status
- Call `sendnotifications` management command
- Observe email being sent

## Checklist:
